extmod/modbluetooth.c

```diff
@@ -785,6 +785,64 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_2(bluetooth_ble_gattc_exchange_mtu_obj, bluetooth
 
 #endif // MICROPY_PY_BLUETOOTH_ENABLE_CENTRAL_MODE
 
+#if MICROPY_PY_BLUETOOTH_ENABLE_L2CAP_COC
+
+STATIC mp_obj_t bluetooth_ble_l2cap_coc_listen(mp_obj_t self_in, mp_obj_t psm_in, mp_obj_t mtu_in) {
+    (void)self_in;
+    mp_int_t psm = mp_obj_get_int(psm_in);
+    mp_int_t mtu = mp_obj_get_int(mtu_in);
+    return bluetooth_handle_errno(mp_bluetooth_l2cap_coc_listen(psm, mtu));
+}
+STATIC MP_DEFINE_CONST_FUN_OBJ_3(bluetooth_ble_l2cap_coc_listen_obj, bluetooth_ble_l2cap_coc_listen);
+
+STATIC mp_obj_t bluetooth_ble_l2cap_coc_connect(size_t n_args, const mp_obj_t *args) {
+    mp_int_t conn_handle = mp_obj_get_int(args[1]);
+    mp_int_t psm = mp_obj_get_int(args[2]);
+    mp_int_t mtu = mp_obj_get_int(args[3]);
+    mp_int_t rxbuf = 0;
+    if (n_args >= 5) {
+        rxbuf = mp_obj_get_int(args[4]);
+    }
+    (void)rxbuf;
```

Comment:
remove this undocumented and unused `rxbuf` arg?

Reply:
Done.